README.md

```diff
@@ -2,9 +2,39 @@
 
 This is the UI component framework for [Yammer.com](https://www.yammer.com/).
 
-It is built with [React](https://reactjs.org/) on top of [Fabric](https://dev.office.com/fabric#/components/) components. Unit tests are run through [Jest](https://facebook.github.io/jest/), isolated development environment and documentation is provided by [Storybook](https://github.com/storybooks/storybook), and visual diff regression is done with [BackstopJS](https://garris.github.io/BackstopJS/).
+It is built with [React](https://reactjs.org/) on top of [Office UI Fabric](https://dev.office.com/fabric#/components/) components. Unit tests are run through [Jest](https://facebook.github.io/jest/), isolated development environment and documentation is provided by [Storybook](https://github.com/storybooks/storybook), and visual diff regression is done with [BackstopJS](https://garris.github.io/BackstopJS/). Visual diff tasks run within a [Docker](https://www.docker.com/) container to ensure consistency between local development environments and CI.
 
-Visual diff tasks run within a [Docker](https://www.docker.com/) container to ensure consistency between local development environments and CI.
+## Using YamUI in your own app
+
+If necessary, add YamUI to your project.
+
+```
```

Comment:
Here you're missing the code block language. It'd probably be ```sh for shell commands.

Reply:
I see. And now the purpose of the linter is clearer.

Reply:
I could have sworn that the error was the same as previous -- that whitespace was necessary. hrm.